### ISSUE TITLE
Extension must be reinstalled when used in conjunction with Microsoft's "Remote - SSH" extention

### ISSUE THREAD

**Issue by kedestin** [robb0wen/synthwave-vscode#136]
I frequently use Microsoft's "Remote - SSH" extention to SSH to various remote hosts. Some extensions (like Python, C++, etc...) must be reinstalled for each host. But others (like color themes), often don't.

This extension previously didn't need to be reinstalled each time, but now it seemingly does. Which makes it exceedingly frustrating, as I often SSH to new remote hosts.

**Comment by Plinpod** [comment]
Yes and it seems like when it forces you to install on Remote - SSH the glow feature is also broken

**Comment by jimsynz** [comment]
and update to this: it doesn't run in either SSH or devcontainer remotes - both of which I use a lot (sometimes both at the same time!). I see the following error on the extension page when using remote containers:

<img width="1219" alt="Screen Shot 2022-07-18 at 10 00 33 AM" src="https://user-images.githubusercontent.com/59449/179426364-f751689f-891b-4fd3-be62-3306bf6c8fbb.png">
.
